src/test/resources/org/eolang/speco/transformations/1-1-coping.yaml

```diff
@@ -0,0 +1,155 @@
+xsl: /org/eolang/speco/1-1-coping.xsl
+
+before: |
+  <?xml version="1.0" encoding="UTF-8"?>
+  <program>
+     <listing>[] &gt; first
```

Comment:
@advasileva I understand that it's just easier to copy already existing sources and past them here (I would definitely do the same), but maybe here is a lot of unnecessary lines? Maybe we can remove redundant parts in order to emphasise what exactely the `1-1-coping.xsl` will do.

Reply:
Good note, I shorted it to:
```xml
before: |
  <?xml version="1.0" encoding="UTF-8"?>
  <program>
     <listing/>
     <errors/>
     <sheets/>
     <objects/>
     <aoi>
        <obj fqn="foo.x">
           <inferred>
              <obj fqn="first"/>
              <obj fqn="second"/>
           </inferred>
        </obj>
     </aoi>
  </program>
```
I can also remove `xml`, `listing`, `errors`, `sheets` and `objects` sections, but the current structure is not very cluttered and allows to quickly understand the transformation within the overall structure of the document.

Reply:
Looks good, it's understandable now what's going on.  No need to remove the structure.